src/pudl/package_data/ferc1/dbf_to_xbrl.csv

```diff
@@ -1161,175 +1161,175 @@ f1_co_directors,45,1994,Line 45,,,,
+f1_comp_balance_db,54,2004,Stores Expense Undistributed (163),reported_value,stores_expense_undistributed,,
+f1_comp_balance_db,55,2004,Gas Stored Underground - Current (164.1),reported_value,gas_stored_current,,
+f1_comp_balance_db,56,2004,Liquefied Natural Gas Stored and Held for Processing (164.2-164.3),reported_value,liquefied_natural_gas_stored_and_held_for_processing,,
+f1_comp_balance_db,57,2004,Prepayments (165),reported_value,prepayments,,
+f1_comp_balance_db,58,2004,Advances for Gas (166-167),reported_value,advances_for_gas,,
+f1_comp_balance_db,59,2004,Interest and Dividends Receivable (171),reported_value,interest_and_dividends_receivable,,
+f1_comp_balance_db,60,2004,Rents Receivable (172),reported_value,rents_receivable,,
+f1_comp_balance_db,61,2004,Accrued Utility Revenues (173),reported_value,accrued_utility_revenues,,
+f1_comp_balance_db,62,2004,Miscellaneous Current and Accrued Assets (174),reported_value,miscellaneous_current_and_accrued_assets,,
+f1_comp_balance_db,63,2004,Derivative Instrument Assets (175),reported_value,derivative_instrument_assets,,
+f1_comp_balance_db,64,2004,(Less) Long-Term Portion of Derivative Instrument Assets (175),reported_value,derivative_instrument_assets_long_term,,This xbrl factoid is mapped to two dbf rows that might have different sign conventions.
+f1_comp_balance_db,65,2004,Derivative Instrument Assets - Hedges (176),reported_value,derivative_instrument_assets_hedges,,
+f1_comp_balance_db,66,2004,(Less) Long-Term Portion of Derivative Instrument Assets - Hedges (176,reported_value,derivative_instrument_assets_hedges_long_term,,This xbrl factoid is mapped to two dbf rows that might have different sign conventions.
+f1_comp_balance_db,67,2004,Total Current and Accrued Assets (Lines 34 through 66),calculated_value,current_and_accrued_assets,,
+f1_comp_balance_db,68,2004,DEFERRED DEBITS,header,,,
+f1_comp_balance_db,69,2004,Unamortized Debt Expenses (181),reported_value,unamortized_debt_expense,,
+f1_comp_balance_db,70,2004,Extraordinary Property Losses (182.1),reported_value,extraordinary_property_losses,,
+f1_comp_balance_db,71,2004,Unrecovered Plant and Regulatory Study Costs (182.2),reported_value,unrecovered_plant_and_regulatory_study_costs,,
+f1_comp_balance_db,72,2004,Other Regulatory Assets (182.3),reported_value,other_regulatory_assets,,
+f1_comp_balance_db,73,2004,Prelim. Survey and Investigation Charges (Electric) (183),reported_value,preliminary_survey_and_investigation_charges,,
+f1_comp_balance_db,74,2004,Preliminary Natural Gas Survey and Investigation Charges 183.1),reported_value,preliminary_natural_gas_survey_and_investigation_charges,TRUE,"This charge was previously combining nat gas and other charged as evidenced by the (183.1, 183.2) and now only showing (183.1). There is no uncombined xbrl factoid for gas even though there is an uncombined xbrl factoid for other."
```

Comment:
Looking at [the XBRL Taxonomy](https://xbrlview.ferc.gov/yeti/resources/yeti-gwt/Yeti.jsp#tax~(id~8*v~72)!con~(id~79785)!net~(a~143*l~35)!lang~(code~en)!path~(g~3709*p~0_0_0_0_3_5)!rg~(rg~4*p~1)) it appears that FERC Account 183.1 is being reported with exactly the same label as this `row_literal` using the ridiculously long name of: `preliminary_natural_gas_survey_and_investigation_charges_and_other_preliminary_survey_and_investigation_charges`

Reply:
Do you know what could have happened to 183.2? I was assuming that the value might be different because it was no longer including 183.2 even though it had the same name, but maybe I shouldn't.

Reply:
Account 183.2 shows up in the XBRL taxonomy with a label of "Other Preliminary Survey and Investigation Charges (183.2)" in association with `other_preliminary_survey_and_investigation_charges`

Reply:
Ok so I did the same thing here as above, marked the newer row literal as `dbf_only = True` instead of the older row literal.

Reply:
It is infuriating and confusing that in the xbrl taxonomy 183.1 is called "Preliminary Natural Gas Survey and Investigation Charges" and 183.2 is called "Other Preliminary Survey and Investigation Charges" and there is an older, combined reporting of both 183.1 and 183.2 that is presumably "Natural Gas and Other Preliminary...." but no, we can't use that as a stand-in row literal because 183.1 is actually called "PreliminaryNaturalGasSurveyAndInvestigationChargesAndOtherPreliminarySurveyAndInvestigationCharges" in properties.... how does that make sense?!?!?! Grrr

Reply:
Anyways, I updated the `dbf_to_xbrl.csv` so that the combined 183.1 and 183.2 records were marked as `dbf_only = True` and gave them the fake factoid name of `preliminary_natural_gas_survey_and_investigation_charges` because `preliminary_natural_gas_survey_and_investigation_charges_and_other_preliminary_survey_and_investigation_charges` was already taken by 183.1 😵‍💫

Reply:
<img width="787" alt="Screen Shot 2022-12-08 at 10 43 47 AM" src="https://user-images.githubusercontent.com/49878195/206525566-d83d21c1-dfa4-4b06-aab1-d577040b2545.png">

Reply:
It doesn't make sense. There are thousands of these categories and someone else probably did an analogous mapping process by hand and made mistakes, or was looking at an old version of the form or something, and so it's riddled with inconsistencies.